eclipse/sw360
# feat(UI): import OSADL obligation information and update screen of Ad…

Signed-off-by: Tran Vu Quan <quan1.tranvu@toshiba.co.jp>

> Please provide a summary of your changes here.
> * Import Obligation information from https://www.osadl.org/
> * New GUI for Adding obligation screen

Issue: https://github.com/eclipse/sw360/issues/513

### How To Test?
> 1. Import Obligation information
> - Open 'Admin Licenses' screen
> - Click 'Import OSADL Information', then click 'Import OSADL license obligations' in the popup dialog (Make sure that you
> already got licenses registered in SW360 system).
> - After importing successfully, open 'Admin Obligations' screen, obligations information from OSADL  will be displayed in the table
> 2. Add obligation screen
> Refer: https://github.com/eclipse/sw360/issues/513#issuecomment-877964654

> Have you implemented any additional tests? No

### Checklist
Must:
